webapp/graphite/render/grammar.py

```diff
@@ -77,7 +77,7 @@
 
 # Metric pattern (aka. pathExpression)
 validMetricChars = ''.join((set(printables) - set(symbols)))
-escapedChar = backslash + Word(symbols, exact=1)
+escapedChar = backslash + Word(symbols + '=', exact=1)
```

Comment:
No, I removed = from the symbols list to allow it to occur in metric names, so it has to be re-added here to avoid breaking existing queries that pass it escaped with a backslash like `\=` rather than as a bare `=`.